libs/aml/src/docs.test.ts

```
@@ -68,6 +64,24 @@
+function listFiles(dir: string): string[] {
+    return fs.readdirSync(dir, {recursive: true}).map(path => pathJoin(dir, path as string))
+}
+
+function getAmlSnippets(path: string, content: string): {snippet: string, version?: string}[] {
+    if (path.indexOf('/demos/') !== -1) return [{snippet: content}] // files in demos are fully AML
+    if (path.endsWith('.aml')) return [{snippet: content}] // aml file
+    if (path.endsWith('.md')) return (content.match(/```aml(v1)?[^]*?```/gm) || []).map(snippet => ({
+        snippet: snippet.replace(/^```aml(v1)?\n/, '').replace(/```$/, ''),
+        version: snippet.startsWith('```amlv1\n') ? 'v1' : undefined
+    }))
+    if (path.endsWith('.html') || path.endsWith('.html.heex')) return (content.match(/<pre><code class="hljs aml(v1)?">[^]*?<\/code><\/pre>/gm) || []).map(snippet => ({
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)